stable/apm-server/templates/deployment.yaml

```diff
@@ -53,6 +53,16 @@ spec:
           - name: http
             containerPort: 8200
             protocol: TCP
+        livenessProbe:
```

Comment:
Can you add these to the daemonset as well?

Reply:
done :+1: